.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -44,9 +47,37 @@ jobs:
           python3 -m pip install -r runtime/bindings/python/iree/runtime/build_requirements.txt
       - name: "Installing requirements"
         run: choco install ccache --yes
+      - name: "Installing MSVC requirements"
+        run: |
+          choco install visualstudio2022buildtools -y
+          choco install visualstudio2022community --package-parameters "--add Microsoft.VisualStudio.Workload.CoreEditor --add Microsoft.VisualStudio.Workload.ManagedDesktop --add Microsoft.VisualStudio.Workload.DesktopDevelopmentWithC++" -y
+          choco install visualstudio2022-workload-nativedesktop -y
       - name: "Configuring MSVC"
-        uses: ilammy/msvc-dev-cmd@0b201ec74fa43914dc39ae48a89fd1d8cb592756 # v1.13.0
-      - name: "Building IREE"
-        run: ./build_tools/cmake/build_all.sh "${BUILD_DIR}"
+        uses: ilammy/msvc-dev-cmd@v1.13.0
+      - name: "Installing iree reqs"
+        run: |
+          choco install cmake
+          choco install ninja
+          choco install Ninja
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+          refreshenv
+      - name: "check space"
+        run: |
+          Get-PSDrive | Format-Table Name, Used, Free
+          Test-Path C:\mnt\azure
+      - name: "Install Bash"
+        run: |
+          $gitPath = "C:\Program Files\Git\bin"
+          $env:PATH += ";C:\Program Files\Git\bin;C:\Program Files\Git\usr\bin"
+          [System.Environment]::GetEnvironmentVariable("PATH", "Process")
+          Test-Path -Path "C:/Program Files/Git/bin/bash.exe"
+      - name: Add Bash to PATH
+        run: |
+          echo "Adding Bash to PATH"
+          echo "C:\Program Files\Git\bin" >> $Env:GITHUB_PATH
```

Comment:
(All of these steps should move to the Dockerfile too. Some amount of workflow scripting is okay if it unblocks these new runners faster... _if the overhead is low enough_)